docs(styles): fast follow update to <code> styles

CITguy opened the pull request:
While I realize that these styles will probably be out of date once design is finished with their work, but in the mean time, this is a fix to make documentation more legible.

### Before
![screen shot 2015-06-23 at 3 43 40 pm](https://cloud.githubusercontent.com/assets/545605/8316898/716eb87a-19bf-11e5-8d70-83d0290468fe.png)

### After
![screen shot 2015-06-23 at 3 44 10 pm](https://cloud.githubusercontent.com/assets/545605/8316906/7745485e-19bf-11e5-8048-5f7132051b13.png)

TheNando commented on demo/assets/demo.less:
```diff
@@ -146,23 +146,30 @@ table.component-attributes {
 }
 
+code {
+    padding: 0px 2px;
```
Most CSS Linters will require that values of 0 not have units specified. We link to a linter in our css styleguide that enforces this.

https://github.com/rackerlabs/encore-ui/blob/master/guides/css-styleguide.md
http://csslint.net/

TheNando commented:
Suggestion regarding unit specification on 0 value properties. Other than that, LGTM.

CITguy commented:
Updated to not apply styles to "Markup", "LESS", "Protractor", etc. Additionally, I removed the generated demo CSS file (I'm not sure why it was added, anyway).

### Before
![screen shot 2015-06-23 at 6 48 00 pm](https://cloud.githubusercontent.com/assets/545605/8319870/7133ce94-19d9-11e5-8bc9-064eb543b71e.png)

### After
![screen shot 2015-06-23 at 6 49 07 pm](https://cloud.githubusercontent.com/assets/545605/8319874/7898254a-19d9-11e5-9e5c-1c7d90880b58.png)

megalord commented:
LGTM
I wouldn't mind seeing this released asap - it's very difficult to read the docs right now.

CITguy commented:
@megalord you and me both :smile:

CITguy commented:
Reverted removal of `generated_demo.css`.

TheNando commented:
LGTM

CITguy commented:
FYI: To be clear, this is a temporary fix to correct readability of docs until official documentation styles are ready.

CITguy commented:
After speaking with @glynnis, part of this was a bug (overriding colors for syntax highlighting), but the original colors were part of the grand re-envisioning of the documentation work that is being done on the design team. I'll be waiting on specific feedback from @glynnis before we do anything else with this PR.

glynnis commented on demo/assets/demo.less:
```diff
@@ -146,23 +146,33 @@ table.component-attributes {
+  code {
+      color: #404040;
+      background-color: #dededd;
+      border: 1px solid #BABABA;
```
Changes to parent elements/scope for these styles is :+1:, however, let's leave colors as is because of more design changes coming down the pipeline (most code blocks will appear on the app grey background instead of white with the docs redesign).

glynnis commented on demo/assets/demo.less:
```diff
@@ -146,23 +146,33 @@ table.component-attributes {
-    border: 0;
-    border-radius: 0;
+p {
+    line-height: 1.5;
```
Line height changes are fine for now re: increasing legibility, but these may be changing shortly

glynnis commented:
Also noting here for future reference the documented process we should try to stick to with PRs w/r/t design involvement and pre-reqs for PRs:
- https://github.com/rackerlabs/encore-ui/blob/master/CONTRIBUTING.md#preparing-an-issue-for-development
- https://github.com/rackerlabs/encore-ui/blob/master/CONTRIBUTING.md#pr-steps

@CITguy and I already spoke and resolved this (and we figured out how to modify this PR so it can be merged :ok_hand: :ok_woman:), but posting here for others who may find this relevant: Many times visual fixes like this that aren't outlined or approved by design upfront can feel like a step backwards for the design team, which has a strategy for rolling out changes (currently handled through sprint goals, etc.). Sometimes the steps to get to the desired outcome are "good enough" and not "great" because of lack of resources and time on the design side.

My hope is that very soon design will have enough bandwidth to begin focusing more on long-term strategy and better communication of that strategy, so that dev has more context and a better understanding of incremental steps necessary for design changes. I'm sure without that context it can seem like design work cuts corners or introduces changes that don't make sense, but a lot of that is due to the volume and breadth of work that we've had to accommodate across multiple projects (which I'm sure has been frustrating for dev!). 

Thanks for bearing with us as we work toward a better process that accounts for vision and improved communication with dev. 

@CITguy let me know if you need any clarification on requested changes. :+1:

CITguy commented:
Reverted to original colors.

CITguy commented:
rebased
